Godeps/_workspace/src/k8s.io/kubernetes/pkg/client/unversioned/clientcmd/loader.go

```diff
@@ -238,7 +237,6 @@ func LoadFromFile(filename string) (*clientcmdapi.Config, error) {
 	if err != nil {
 		return nil, err
 	}
-	glog.V(3).Infoln("Config loaded from file", filename)
```

Comment:
@deads2k I dunno under which loglevel we really run the extended tests, but I can see that message even if I set it glog.V(5).Infof(). 

agree it is useful for "regular use". Maybe setting the loglevel for "oc" command used in extended tests to 0 will make it better.